src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Update to the best version of Explore</li>
+      <li>Chat requires Microsoft Edge v.14 or better</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### Help Center
+
+Capitalize both words. Also, refer to it as Help Center, not the Help Center.
+
+Example: "Edit articles in Help Center."
+
+A possessive pronoun is okay.
+
+Example: “Edit articles in your Help Center.”
+
+### help desk
+
+Zendesk makes customer service software, not help desk software. The term help
+desk should only be used when specifically addressing IT or help desk issues.
+
+In most cases, you can use Zendesk Support instead.
+
+Example: "You can use Zendesk Support to improve your relationship with your
+customers."
+
+### hint
+
+Don’t use it as a heading for a type of note. Use tip instead.
+
+<!-- markdownlint-disable -->
+
+### i.e.
+
+Abbreviation for id est, meaning “that is.” Use that is or in other words
+instead.
+
+### incoming, outgoing
+
+Use to refer to email messages that are being downloaded or being sent. Do not
+use inbound and outbound unless you have no other choice.
+
+### Insights
+
+Capitalize when referring to the Zendesk product.
+
+### JavaScript
+
+Use [Pascal case](https://en.wikipedia.org/wiki/Camel_case). Not Javascript or javascript.
+
+### knowledge base
+
+Two words, lowercase. Sometimes called KB for short.
+
+### Learn about vs. learn more
+
+We reserve the linked words "Learn about" for links to the Zendesk Help Center.
+Avoid "learn more" to prevent the repetition of the phrase for screen readers.
+Include detail about what the user will learn.
+
+- Use this formula: "Learn about [linked subject]”
+- Aim for a maximum of four words per link
+- Avoid Help Center article names that can’t be localized
+
+<BestPractice>
+  <Do>
+    <p>Learn about user profiles</p>
+  </Do>
+  <Dont>
+    <p>Learn more</p>
+  </Dont>
+</BestPractice>
+
+### macOS
+
+Use when referring to the operating system. Don't use Mac OS.
+
+### min, max
+
+It’s better to spell these out in full (“minimum” or “maximum”) but the short
+form is OK when you’re short on space. You don’t need a period after min or max.
+
+### Net Promoter Score℠, NPS®
+
+Uppercase. Use trademark symbols on first reference. Net Promoter Score is a
+service mark (℠); NPS is a registered trademark (®). Symbols should be placed
+directly after the mark, in superscript, and slightly smaller than the mark
+text.
+
+### new
+
+New requires gender for many languages. Add is a better substitute. Features
+that are new to the user are best communicated through emails, landing pages,
+and marketing materials.
+
+### OK
+
+Avoid OK as a button label. Button labels should make it clear what’s going to
+happen when the user clicks it.
+
+On mobile devices it’s permitted to use OK for button text.
+
+### OAuth
+
+Not oAuth.
+
+### on, onto
+
+You record conversations on a ticket. Onto conveys actual motion. For example, I
```

Comment:
You record conversations _on_ a ticket rather than onto. Onto conveys actual motion. For example, I jumped onto the parallel bars. If you’re confused, just say "up" before _on_ in the sentence. If it makes sense, onto is the right word.